pkg/util/test/util.go

```diff
@@ -2852,6 +2854,10 @@ func validateStorkSecurityEnvVar(cluster *corev1.StorageCluster, storkDeployment
 		return err
 	}
 
+	if err := ValidateGrafana(cluster); err != nil {
```

Comment:
Why are we validating this as a part of `validateStorkSecurityEnvVar()` function?